lib/active_fedora/rdf/premis.rb

```diff
@@ -0,0 +1,6 @@
+require 'rdf'
+module ActiveFedora::RDF
+  class Premis < RDF::StrictVocabulary("http://www.loc.gov/premis/rdf/v1#")
```

Comment:
Can we use RDF::Vocab::Premis.hasMessageDigest (https://github.com/ruby-rdf/rdf-vocab/blob/develop/lib/rdf/vocab/premis.rb#L742) rather than defining our own vocabulary?